x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/ranges/advanced_editor.tsx

```diff
@@ -159,6 +164,25 @@ export const RangePopover = ({
           </EuiFlexItem>
         </EuiFlexGroup>
       </EuiFormRow>
+      <EuiFormRow>
+        <LabelInput
+          value={label || ''}
+          onChange={(newLabel) => {
+            const newRange = {
+              ...tempRange,
+              label: newLabel,
```

Comment:
So this resets to the empty string to indicate that it's empty, is that going to cause problems with equality? Imagine the steps 1. Create custom label 2. Delete custom label 3. Formats correctly?

Reply:
Tested, this doesn't cause any issues